api_v3/lib/types/partner/KalturaPartner.php

```diff
@@ -429,6 +460,13 @@ public function fromPartner(Partner $partner)
 	
 	public function doFromObject($partner, KalturaDetachedResponseProfile $responseProfile = null)
 	{
+		$passwordValidation = $partner->getPasswordStructureValidations();
+		if(isset($passwordValidation[0]))
+		{
+			$this->passwordStructureValidations = $passwordValidation[0][0];
+			$this->passwordStructureValidationsDescription = $passwordValidation[0][1];
+		}
+		
 		parent::doFromObject($partner);
 		
```

Comment:
After the doFromObject
need to take the values of $this->passwordStructureValidations from $partner->getPasswordStructureRegex
$this->passwordStructureValidationsDescription from $partner->getInvalidPasswordStructureMessage

if the above is empty, need to take the info from $regexps = kConf::get('user_login_password_structure') and $msg =kConf::get(' invalid_password_structure_message')